events/tests.py

```diff
@@ -32,6 +101,19 @@ def test_get(self):
 
         self.assertEqual(r.status_code, 200)
 
+    def test_get_sets_billable_to_false(self):
```

Comment:
unclear title (and next test).  suggest test_new_event_billable_default_is_false_when_previous_event_is_false despite the long name
and test_new_event_billable_default_is_true